docs/deployment/deployK8s-AWS.md

```diff
@@ -5,343 +5,158 @@ title: Prepare AWS EKS for Multi-Region Deployments
+4.  Accept transit gateway peering attachment.
 
-      * At the top right of the page, change the region to **Canada Central**.
+    * Once the transit gateway peering connection shows `pending acceptance` as its **State**, toggle to the `us-east-1` region and select **Transit gateway attachments**.
 
-      * Select the route table you recorded for the `ca-central-1` (Canada Central) VPC, and click the `Routes` button.
+    * You should see the attachment you just made. Select **Actions** and click accept.
 
-         Two routes are displayed.
+    * Add a name to this attachment such as `demo-peering-attachment-eu-west-1`.
 
-      * Click **Edit Routes** --> **Add Route**, and for **Destination**, enter the CIDR block for the `us-west-2` cluster (10.0.0.0/16).
+        > Make note that this name is refering to the region it is peering to, not the region it is being create in.
 
-      * For **Target**, select the VPC peering connection you created in a prior step. Click **Save Routes**.
+5. Attach VPCs to the transit gateways in each region. Toggle to the `eu-west-1` region.
 
-        A route for the `us-west-2` cluster directed to the peering connection is displayed.
+    * Navigate to the **Transit gateway attachments** section and click **Create transit gateway attachments**.
 
-      * If more than one routing table is used for the `ca-central-1` VPC, repeat the previous steps for each routing table.
+    * Add a name tag such as `demo-vpc-eu-west-1`.
 
-8. Update the Security Groups for each VPC. Note the Security Group IDs for each VPC and add inbound and outbound rules for both the `us-west-2` VPC, and the `ca-central-1` VPC.
+    * Select the Transit gateway id that you just made in the `eu-west-1` region.
 
-      * In the VPC Dashboard, select **Security Groups** for the Canada Central region.
+    * Select the Vpc Id that you made note of in step 3 for the `eu-west-1` region.
 
-      * Apply a filter to find the security groups for the `ca-central-1` cluster, and select the security group with “-nodegroup” in the name.  This security group is used for the firewall settings for all the worker nodes in the `ca-central-1` cluster.
+    * Click **Create transit gateway attachment**.
 
-      * Click **Inbound Rules** --> **Add Rule**.
+    * Repeat this step in `us-east-1` region.
 
-      * Select these values for the rule:
+6.  Accept transit gateway VPC attachments. Toggle to the `eu-west-1` region.
 
-         * Type:  Custom TCP Rule
+    * Navigate to the **Transit gateway attachments** section and click **Create transit gateway attachments**.
 
-         * Protocol: TCP
+    * You should see the vpc attachment you just made. Select **Actions** and click accept.
 
-         * Port Range: 7600-7700
+        > Note if you are using different accounts to create transit gateways and their attachments, then the name tag will  not be visable here. If thats the case you should add an attachment name now, such as `demo-vpc-eu-west-1`.
```

Comment:
**_Note: If you_** are using different accounts to create transit gateways and their attachments, **_the name tag will not be visible_** here. In this situation, you should add ...